GDPR-Phase2-Java/src/main/java/com/gdpr/phase2/model/SortedVector.java

```diff
@@ -0,0 +1,39 @@
+package com.gdpr.phase2.model;
+
+import java.util.*;
+
+
+/**
+ * @Description
+ * Its a java replacement of the "Vector" class defined in the Typescript code of Chris's Library
+ */
+public class SortedVector {
+    public int bitLength;
+    public SortedSet<Integer> set;
```

Comment:
I think we shouldn't expose these properties if we are providing accessor functions

Reply:
Right i will make the changes.